book/07-git-tools/sections/stashing-cleaning.asc

```diff
@@ -193,52 +193,52 @@ I-stash ang hunk na ito [y,n,q,a,d,/,e,?]? y
 Naka-save na gumaganang direktoryo at estado ng indeks WIP sa master: 1b65b17 idinagdag ang indeks na file
 ----
 
-==== Creating a Branch from a Stash
+==== Paglilikha ng isang Branch mula sa isang Stash
 
-If you stash some work, leave it there for a while, and continue on the branch from which you stashed the work, you may have a problem reapplying the work.
-If the apply tries to modify a file that you’ve since modified, you’ll get a merge conflict and will have to try to resolve it.
-If you want an easier way to test the stashed changes again, you can run `git stash branch <branch>`, which creates a new branch for you with your selected branch name, checks out the commit you were on when you stashed your work, reapplies your work there, and then drops the stash if it applies successfully:
+Kung ikaw ay nag-stash ng ilang trabaho, iwanan ito sa ilang saglit, at magpatuloy sa branch kung saan ikaw nag-stash ng trabaho, maaari kang magkaroon ng isang problema sa muling paglalapat ng trabaho.
+Kung sinusubukan ng paglalapat na baguhin ang isang file na binago mo na, makakakuha ka ng salungatan sa pagsasama-sama at kakailanganin mong lutasin ito.
+Kung gusto ang isang mas madaling paraan upang suriing muli ang mga naka-stash na pagbabago, maaari mong patakbuhin ang `git stash branch <branch>`, kung saan ay lumilikha ng isang bagong branch para sa iyo kasama ang iyong napiling pangalan ng branch, nagsusuri sa iyong commit kapag ikaw ay nag-stash ng iyong trabaho, muling naglalapat sa iyong trabaho doon, at pagkatapos ay naghuhulog sa stash kung ito ay matagumpay na nailalapat:
```

Comment:
para sa iyo kasama ang iyong napiling pangalan ng branch to para sa iyong napiling pangalan ng branch